docs/0.3/using_purchase_order.md

```diff
@@ -230,102 +214,70 @@ public key file in the next command.
     gnrl $(cat ~/.grid/keys/beta-agent.pub) \
     --active \
     --role admin \
-    --role po-manager`
+    --role po-manager
     ```
 
 ### Create Purchase Orders
 
 {:start=1}
 
 1.  Using the `gridd-beta` command line, propose a draft purchase order on behalf
-    of Fred's Farm.
+    of MyOrg.
 
     ```
     root@gridd-beta:/# grid po create \
-    --org farm \
-    --id po_number: 0123456 \
-    --workflow-status Issued \
+    --buyer-org gnrl \
+    --seller-org myorg \
+    --alternate-id po_number:0123456 \
+    --workflow-state create \
+    --workflow-id built-in::collaborative::v1 \
     --wait 100
 
     root@gridd-beta:/# grid po version create \
-    --org farm \
-    --po 0123456 \
+    po_number:0123456 \
+    1 \
     --order-xml <path to order> \
     --draft \
-    --workflow-status Editable \
-    --wait 100 \
-    v1
-    ```
-
-1.  Using the `gridd-alpha` command line, accept the proposed purchase order
-
-    ```
-    root@gridd-alpha:/# grid po version update \
-    --org farm \
-    --po po_number:0123456 \
-    --not-draft \
-    --workflow-status Proposed \
-    --wait 100 \
-    v1
-
-    root@gridd-alpha:/# grid po update \
-    --org farm \
-    --workflow-status Issued \
-    --accepted-version v1 \
+    --workflow-state create \
     --wait 100
-    ```
-
-1.  Using the `gridd-beta` command line, accept the purchase order
-
-    ```
-    root@gridd-beta:/# grid po version update \
-    --org=farm \
-    --po po_number:0123456 \
-    --not-draft \
-    --workflow-status Accepted \
-    --wait=100 \
-    v1
 
     root@gridd-beta:/# grid po update \
-    --org=farm \
-    --workflow-status Confirmed \
-    --accepted-version v1 \
-    --wait=100
+    po_number:0123456 \
+    --workflow-state issued \
+    --wait 100
     ```
 
-1.  Using the `gridd-alpha` command line, update the purchase order
+1.  At the same time, MyOrganization is creating a version of their own. Using
+    the `gridd-alpha` command line, create a version for the purchase order
 
     ```
-    root@gridd-alpha:/# grid po version update \
-    --org=farm \
-    --po po_number:0123456 \
-    --order-xml <path to updated order> \
+    root@gridd-alpha:/#grid po version create \
```

Comment:
Should have a space before `grid po version ...`

Reply:
updated